### PR TITLE
Adding basic test app

### PR DESCRIPTION
**Pull request by kiwiupover**
Adding `basic-app` to the test-packages to represent a basic ember app that is fastbooted using `ember-cli-fastboot`
Most of the new test will be against the basic-app.

This PR also replaces  `ember-cli-fastboot/test/build-test.js` with https://github.com/ember-fastboot/ember-cli-fastboot/pull/775/files#diff-518e8f29120d406a1931ce09ccca5d27

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,13 +19,16 @@ on:
   - cron:  '0 6 * * 0'
 
 jobs:
-  test-mocha:
+  test-legacy-mocha:
```
Comment (kiwiupover) [inline .github/workflows/ci.yml:22]: For you @mansona mate.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Ember FastBoot
 
 [![npm version](https://badge.fury.io/js/ember-cli-fastboot.svg)](https://badge.fury.io/js/ember-cli-fastboot)
-[![Actions Status](hhttps://github.com/ember-fastboot/ember-cli-fastboot/workflows/CI/badge.svg)](https://github.com/ember-fastboot/ember-cli-fastboot/actions)
+[![Actions Status](https://github.com/ember-fastboot/ember-cli-fastboot/workflows/CI/badge.svg)](https://github.com/ember-fastboot/ember-cli-fastboot/actions)
```
Comment (kiwiupover) [inline README.md:4]: This should be working now.

File: test-packages/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+### Test Packages App
+
+- [Basic App](./test-packages/basic-app) This basic ember app that is fastbooted using ember-cli-fastboot. Most test will be against this basic-app. When an app needs custom ember-cli-fastboot setup another ember app should be added to the test-packages folder.
```
Comment (kiwiupover) [inline test-packages/README.md:3]: @rwjblue is there you wanted this readme?

File: test-packages/basic-app/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,11636 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (rwjblue) [inline test-packages/basic-app/yarn.lock:1]: I think we probably don't actually want to  have a second `yarn.lock` here.

**Comment by kiwiupover** [comment]
@rwjblue the legacy mocha tests are passing after switching to use `volta` to install npm 4

**Pull request merged**